### PR TITLE
Collijk/feature/use GitHub actions for ci

### PR DESCRIPTION
**Pull request by collijk**
Delete old docs, swap from travis ci (which doesn't run anymore) to github actions for build and test.

### CODE REVIEW

**Comment by miker985** [comment]
image

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (miker985) [inline .github/workflows/build.yml:7]: is centos not an option?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    strategy:
+      matrix:
+        python-version: [3.7] #, 3.8, 3.9]
```
Comment (miker985) [inline .github/workflows/build.yml:10]: is this for speed of execution or does the code not currently run on 3.8/9?

**Review by miker985 — APPROVED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (collijk) [reply .github/workflows/build.yml:10]: It was for testing.  I should put these back.

File: .github/workflows/build.yml (hunk 1 above)
Comment (collijk) [reply .github/workflows/build.yml:7]: Unfortunately, no.

**Pull request merged**